uint32   timestamp
int32    encoder_left
int32    encoder_right
float32  steering_angle